tests/e2e/helpers/base.js

```diff
@@ -2,6 +2,36 @@ import { ClientFunction, Selector } from 'testcafe';
 
 const PAGE_FLIP_WAIT_TIME = 1000;
 
+const getHash = ClientFunction(() => document.location.hash);
+const getUrl = ClientFunction(() => window.location.href);
+
+/**
+ * Check URL page paramter in # and path
+ */
+const expectPage = ClientFunction(() => {
+  const hash = document.location.hash;
+  console.log(hash);
+  if (hash) {
+    return hash.indexOf('#page/') > -1;
+  } else {
+    return window.location.href.indexOf('/page/') > -1;
+  }
+});
+
+/**
+ * Check URL mode paramter in # and path
+ *
+ * @param mode '1up', '2up', 'thumb'
+ */
+const expectMode = ClientFunction((mode) => {
```

Comment:
The return of this function can lead to a false positive when asserting with `.ok()`, as it's looking for any truthy value and `-1` is truthy in JS.

Reply:
fixed